src/OpenSSL/SSL.py

```diff
@@ -223,11 +223,8 @@
 except AttributeError:
     pass
 
-try:
-    OP_LEGACY_SERVER_CONNECT = _lib.SSL_OP_LEGACY_SERVER_CONNECT
-    __all__.append("OP_LEGACY_SERVER_CONNECT")
-except AttributeError:
-    pass
+OP_LEGACY_SERVER_CONNECT = _lib.SSL_OP_LEGACY_SERVER_CONNECT
+__all__.append("OP_LEGACY_SERVER_CONNECT")
```

Comment:
If this is no longer conditional, wouldn't it be better to move it up with the other definitions above and include it straight into `__all__`? This will need a rebase since I added an `int` type annotation, sorry :)